models/staging/graph/stg_columns.sql

```diff
@@ -0,0 +1,11 @@
+with
+
+final as (
+
+    {{ dbt_utils.union_relations([
+        ref('base_node_columns'),
+        ref('base_source_columns')
+    ])}}
+)
+
+select distinct * from final
```

Comment:
`dbt_utils.union_relations` adds synthetic column `_dbt_source_relation`. This makes `select distinct *` ineffective (i.e. it will produce the same result as `select *`).